src/cli/README.md

```diff
@@ -269,6 +270,55 @@ Set the Thread Child Timeout value.
 Done
 ```
 
+### coap server \<phase\>
+
+Starts and stops the simple CoAP server.
+
+* phase: Either "start" or "stop" the server (mandatory).
+
+```bash
+> coap server start
+Server started with resource '': Done
+> coap server stop
+Server stopped: Done
+```
+
+### coap server name \<URI\>
+
+Outputs the currently used URI String of the CoAP resource.
+
+* URI: If provided the URI String will be changed to the new value (optional).
+
+```bash
+> coap server name
+Current resource name is '': Done
+> coap server name test
+Changing resource name to 'test': Done
+```
+
+### coap client \<method\> \<IPv6address\> \<URI\> \<payload\> \<messageType\>
+
+Simple CoAP client that can send Non-/Confirmable GET/PUT/POST/DELETE messages.
+
+* method: CoAP method to be used (GET/PUT/POST/DELETE) (mandatory).
+* IPv6address: IP address of the CoAP server to query (mandatory).
+* URI: URI String of the resource on the CoAP server (mandatory).
+* payload: In case of PUT/POST/DELETE a payload can be encapsulated (optional).
```

Comment:
I might be wrong but i believe you can use this brackets: "[" and "]" to indicate that parameter is optional e.g. `[payload]` instead of `<payload>`